airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json

```diff
@@ -42,6 +42,95 @@
           }
         }
       }
+    },
+    "lastUpdatedByUserId": {
+      "type" : ["null", "integer"]
+    },
+    "contactCounts": {
+      "properties" : {
+        "active": {
+          "type" : ["null", "integer"]
+        },
+        "enrolled": {
+          "type" : ["null", "integer"]
+        }
+      }
+    },
+    "description": {
+      "type" : ["null", "string"]
+    },
+    "originalAuthorUserId": {
+      "type" : ["null", "integer"]
+    },
+    "migrationStatus": {
+      "properties" : {
+        "enrollmentMigrationStatus": {
+          "type" : ["null", "string"]
+        },
+        "enrollmentMigrationTimestamp": {
+          "type" : ["null", "integer"]
+        },
+        "flowId": {
+          "type" : ["null", "integer"]
+        },
+        "lastSuccessfulMigrationTimestamp": {
+          "type" : ["null", "integer"]
+        },
+        "migrationStatus": {
+          "type" : ["null", "string"]
+        },
+        "platformOwnsActions": {
+          "type" : ["null", "boolean"]
+        },
+        "portalId": {
+          "type" : ["null", "integer"]
+        },
+        "workflowId": {
+          "type" : ["null", "integer"]
+        }
+      }
+    },
+    "updateSource": {
+      "properties" : {
+        "sourceApplication": {
+          "properties" : {
+          "serviceName": {"type" : ["null", "string"]},
+          "source": {"type" : ["null", "string"]}
+          }
+        },
+        "updatedAt": {"type" : ["null", "integer"]},
+        "updatedByUser": {
+          "properties" : {
+            "userEmail": {"type" : ["null", "string"]},
+            "userId": {"type" : ["null", "integer"]}
+          }
+        }
+      }
+    },
+    "creationSource": {
```

Comment:
`type` field is missing here as we as in many other properties. I don't think this is a valid definition

Reply:
good point, I'll go through schemas once again to see if some type declarations were missed

Reply:
updated type declarations here and in other schemas